bower.json

```diff
@@ -33,6 +33,7 @@
     "leaflet": "^1.0.3",
     "leaflet.markercluster": "^1.0.3",
     "leaflet-bing-tiles": "^1.0.0",
+    "leaflet-google-tiles": "https://github.com/ge-smallworld/leaflet-google-layer.git",
```

Comment:
Please give a version range for this. This ensures that if you need to do a major upgrade that breaks backward API compat we can decide when to opt-in to it. Also preferred to reference github repos with the following format in bower.json, so the downstream user can decide if they'll use ssh or https to clone with bower:

```
"leaflet-google-tiles": "ge-smallworld/leaflet-google-layer#^1.0.0"
```